documentation/versioned_docs/version-cloud/_constants.md

```
@@ -0,0 +1,6 @@
+{/* This file stores constants used across the documentation */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)